Package.swift

```diff
@@ -38,3 +38,9 @@ let package = Package(
         ),
     ]
 )
+
+for target in package.targets {
+    var settings = target.swiftSettings ?? []
+    settings.append(.enableExperimentalFeature("StrictConcurrency=complete"))
+    target.swiftSettings = settings
+}
```

Comment:
Should we just inline this in the targets?

Reply:
I've been using the same snippet for all packages we're working through. Is it worth diverging here?